target/linux/ramips/mt7620/base-files/etc/board.d/03_gpio_switches

```diff
@@ -22,6 +22,9 @@ head-weblink,hdrm200)
 	ucidef_add_gpio_switch "io4" "I/O 4" "14"
 	ucidef_add_gpio_switch "power_mpcie" "mPCIe power" "21" "1"
 	;;
+lb-link,bl-w1200)
+	ucidef_add_gpio_switch "led_blackout" "ETH LEDs blackout" "10" "1"
```

Comment:
Hmm, if this is called "blackout", I'd expect the LEDs to be off when set to "1", i.e. by default. If LEDs are on for 1=enabled, one should change the name.